massgov/mayflower-patternlab
# DP-8854 Footer logo

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

The footer portion which is set up as `03-organisms/by-template/footer.twig` has been hardcoded in the layout template, `page.html.twig`, in Drupal and not using Mayflower where it could be used. This has been causing some inefficiency for maintenance.

In this PR, the following are done:

- Update the content in `<footer>` to match Drupal:
     - Add an aria role to `<footer>` tag
     - Add the privacy policy link
     - Change the hard-coded copyright year to the twig markup to print the current year
- Updates the Google translate JS in the template to match one in the Drupal
- Add a twig block for the navigation to set up the both implementations for Mayflower and Drupal, so the navigation is rendered properly in both environments.

Associated Drupal PR: [DP-8854 Footer #2290](https://github.com/massgov/mass/pull/2290)
If this PR is released before Drupal one, it doesn't affect Drupal side since it has its own hard-coded markup in its themeing.  However, releasing Drupal change without the Mayflower change would cause Releasing this Drupal change without the Mayflower change would cause the footer navigation to be missing.

## Related Issue / Ticket

- [DP-8854 Footer logo](https://jira.state.ma.us/browse/DP-8854)

## Steps to Test


1. Go to any pages with the footer.
2. Check for any difference for the footer display against the prod.
3. Check the copyright year is current.
4. Check the navigation links work as they are supposed to.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:


#### Impacted Areas in Application
